docs/src/main/sphinx/release/release-368.md

```diff
@@ -0,0 +1,45 @@
+# Release 368 (xx Jan 2022)
+
+## General
+
+* Improve performance of arithmetic operations involving decimals with precision
+  larger than 18. ({issue}`10051`)
+* Introduce `prioritize-utilization` query execution policy that aims to improve
+  resource utilization for queries that involve joins and for concurrent
+  workloads. ({issue}`10350`)
+* Fix delegated refresh materialized views queries not transitioning into
+  `RUNNING` state. ({issue}`10360`)
+* Allow setting per task memory limits via `query.max-total-memory-per-task`
+  config property or via `query_max_total_memory_per_task` session property.
+  ({issue}`10308`)
+
+## Hive connector
+
+* Improve performance of decoding decimal values with precision larger than 18
+  in ORC, Parquet and RCFile data. ({issue}`10051`)
+
+## Iceberg connector
+
+* Fix Iceberg table creation with location when schema location inaccessible.
+  ({issue}`9732`)
+
+## MemSQL connector
+
+* Add support for `time` type. ({issue}`10332`)
+
+## Oracle connector
+
+* Fix incorrect result when a `date` value is older than or equal to
+  `1582-10-14`. ({issue}`10380`)
+
+## SQL Server connector
+
+* Update JDBC driver to version `9.4.1.jre8`. ({issue}`10465`)
```

Comment:
fyi @ebyhr

Reply:
@mosabua I don't think we mentioned JDBC driver upgrade in the past release notes.

Reply:
Indeed. We should only describe user-visible effects from this upgrade, if any.

Reply:
Did you verify there are no user facing changes (e.g. compatibility)? I did not but thought this is an important update worth mention. Should I check? Also .. why are we using the jre8 one anyway?

https://docs.microsoft.com/en-us/sql/connect/jdbc/download-microsoft-jdbc-driver-for-sql-server?view=sql-server-ver15

<dependency>
    <groupId>com.microsoft.sqlserver</groupId>
    <artifactId>mssql-jdbc</artifactId>
    <version>9.4.1.jre11</version>
</dependency>

Should I send a PR for that?

Reply:
https://docs.microsoft.com/en-us/sql/connect/jdbc/release-notes-for-the-jdbc-driver?view=sql-server-ver15

Reply:
Should I send a PR for the update @martint @ebyhr ?

Reply:
Not sure what's the difference between the `.jre8` and `.jre11` and why they would publish different artifacts, but we should be using whichever version is compatible with both Java 11 and Java 17, as we'll be moving to that version of the JDK soon.

Reply:
compile target mostly when looking at the code .. so it might be better to use 11 .. I will send a PR

Reply:
Alos.. Java 17 support is only in preview with the new 10.x version of the driver

https://search.maven.org/artifact/com.microsoft.sqlserver/mssql-jdbc

Reply:
Here you go .. lets see if that passes CI ... https://github.com/trinodb/trino/pull/10495

Reply:
Looks like it passed CI at least.. I will leave the rest to you @ebyhr

Reply:
That change got merged now.